deploy.sh

```diff
@@ -11,7 +11,8 @@ LOG_FILE=deployment.log
 #   PORT=8100 OPENFISCA_PORT=12100 ./deploy.sh aah
 # Example for production:
 #   PORT=8000 OPENFISCA_PORT=2000 PUBLIC_HOST=mes-aides.gouv.fr PROTOCOL=https ./deploy.sh
-TARGET_BRANCH=${1:-master}
+USER=`whoami`
+TARGET_BRANCH=${1:-${USER##mes-aides-}}
```

Comment:
You probably don't need the double `##` (longest-match), a single one should suffice.

Do we still need to take `$1` as a parameter then? Sounds like a recipy for wasting an hour trying to understand that user `mes-aides-herp` actually deployed branch `derp`.